x-pack/plugins/enterprise_search/public/applications/app_search/components/search_ui/types.ts

```diff
@@ -10,5 +10,6 @@ export enum ActiveField {
   Filter = 'Filter',
   Sort = 'Sort',
   Url = 'Url',
+  Thumb = 'Thumb',
```

Comment:
[optional/not a blocker] Might suggest `'Thumbnail'` just for consistency

Reply:
OH wait, I see, it needs to match the SVG class. Wonder if we should update the SVG then, or if you could do

```suggestion
  Thumbnail = 'Thumb',
```

ehh it's probably fine